man/fabtests.7.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: page
+title: fabtests(7)
+tagline: Fabtests Programmer's Manual
+---
+
+## NAME
+
+Fabtests
+
+##SYNOPSIS
+
+Fabtests is a set of examples for fabric providers that demonstrates various features of libfabric- high-performance fabric software library.
+
+## OVERVIEW
+  
+Libfabric defines sets of interface that fabric providers can support. The purpose of Fabtests examples is to demonstrate some of the major features. The goal is to familiarize users with different functionalities libfabric offers and how to use them.
+
+The tests are divided into four categories:
+
+### Simple
```

Comment:
I thought we were using two different levels of header. Do you prefer all to be the same level?